com_languagepack/site/models/newpack.php

```diff
@@ -265,14 +265,24 @@ public function save($data)
 			'version'     => $completeVersion,
 			'alias'       => str_replace('.', '-', $completeVersion),
 			'maturity'    => 'stable',
-			'description' => '<p>This is the ' . $languageName . ' Language Pack for Joomla! ' . $joomlaVersion . '</p>',
+			'description' => '<p>This is the ' . $languageName . ' Language Pack for Joomla! ' . $joomlaVersion,
 			'created'     => $dateNow->toSql(),
 			'access'      => '1',
 		];
 
+		// TODO: Check if this is a string or integer in the data and delete one of these.
+		if ($releaseVersion === 1 || $releaseVersion === '1')
+		{
+			$arsReleaseData['description'] .= '</p>';
+		}
+		else
+		{
+			$arsReleaseData['description'] .= ' (v' . $releaseVersion . ')</p>';
```

Comment:
hmm I see, still wonder why the p tags are here.

Reply:
This is what renders in the ARS View in the HTML Interface